FAQUse: update History (#1254)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

:::details
### How can I check the transactions history?

> The wallet's main page displays the history of all transactions and coinjoins made with this specific wallet.
> It includes:
> - BTC being being received 
> - BTC being sent out of the wallet, meaning the payment to an external address and the fee.
> - coinjoins & the it's corresponding outgoing fee

Reviewer: I would not mention fees, otherwise we can mention the labels or the balance...

```suggestion
The wallet's main page displays the history of all transactions made with this specific wallet.
It includes receiving, sending, and coinjoin transactions.
```
